### PR TITLE
feat: module 101-02 content

### PR DESCRIPTION
**Pull request by bguiz**
- added a student guide for module 101-02
- added MCQs for module 101-02
- added a graded assessment for module 101-02

### CODE REVIEW

File: hdc10102GradedAssessment/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+and you will need to make a fresh copy of it within your own git repository,
+and use that repo for submissions.
+You **should not** work directly within this repo (it will not graded).
+
+If you have yet to initialise a repo for assessments submissions,
+perform the following steps:
+
+```shell
+mkdir -p ../hedera-dev-course-101-graded-assessments
+git cd hedera-dev-course-101-graded-assessments
```
Comment (michielmulders) [inline hdc10102GradedAssessment/README.md]: Is this command correct? the `git cd`?

File: hdc10102GradedAssessment/script-hdc10102Task03.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+https://
```
Comment (michielmulders) [inline hdc10102GradedAssessment/script-hdc10102Task03.txt:1]: aren't we missing the hashscan url here for account 0.0.10?

**Review by michielmulders — COMMENTED** [review]
thanks for adding the templates as well. I spotted the missing URL for hashscan grade assessment result and perhaps a typo in the command `git cd`?

File: hdc10102GradedAssessment/script-hdc10102Task03.txt (hunk 2 above)
Comment (bguiz) [reply hdc10102GradedAssessment/script-hdc10102Task03.txt:1]: This is intentional, this is the version that is incomplete, and the student is to replace it with the full URL. (so this serves as more of a hint)

Comment (michielmulders) [reply hdc10102GradedAssessment/script-hdc10102Task03.txt:1]: A/h I see!

**Review by michielmulders — APPROVED** [review]

**Pull request merged**